### DISCUSSION TITLE
Angular: applicationConfig Type has moved?

### DISCUSSION THREAD

**Discussion by smlimasu** [storybookjs/storybook#28137] [Help]
### Summary

When I looked into https://github.com/storybookjs/storybook/blob/next/code%2Fframeworks%2Fangular%2Fsrc%2Fclient%2Ftypes.ts I am seeing that applicationConfig has moved from angular/platform-browser to angular/core. 

I am wondering if that import path should be adjusted?

**Comment by Marklb** [comment]
~~That looks correct. I just checked Angular's changelog and it looks like that was moved in v16:~~ https://github.com/angular/angular/blob/main/CHANGELOG.md#deprecations-2 *It was clarified why not yet in another comment.*

**Comment by valentinpalkovic** [comment]
As long as we support Angular 15, we can’t adjust the type import. With Storybook 9 we will drop Angular 15 support and then we will also be able to adjust the import.
